src/components/views/rooms/wysiwyg_composer/hooks/usePlainTextListeners.ts

```diff
@@ -33,20 +37,43 @@ function amendInnerHtml(text: string): string {
         .replace(/<\/div>/g, "");
 }
 
+/**
+ * Generates all of the listeners and the ref to be attached to the editor. Also returns
+ * pieces of state and utility functions that are required for use in other hooks and by
+ * the autocomplete component.
+ *
+ * @param initialContent - can set the content of the editor on mount
+ * @param onChange - called whenever there is change in the editor content
+ * @param onSend - called whenever the user sends the message
+ * @returns
+ * - `ref`: a ref object which the caller must attach to the HTML `div` node for the editor
+ * * `autocompleteRef`: a ref object which the caller must attach to the autocomplete component
+ * - `content`: state representing the editor's current text content
+ * - `setContent`: the setter for the content state
```

Comment:
```suggestion
 * - `setContent`: the setter function for `content`.
```

Reply:
(Maybe in a future PR we could move the definition of `content` up to the caller and just pass in `setContent` to this hook)

Reply:
Yep agree - the next chunk of work in this section to add mentions (started, but WIP) has already highlighted and changed this.